ballista-examples/README.md

```diff
@@ -47,9 +47,6 @@ RUST_LOG=info ./target/release/ballista-executor -c 4
 
 ## Running the examples
 
-Refer to the instructions in [DEVELOPERS.md](../DEVELOPERS.md) to define the `ARROW_TEST_DATA` and
```

Comment:
https://github.com/apache/arrow-datafusion/blob/master/CONTRIBUTING.md#bootstrap-environment 
Can the relevant content in this file be deleted as well?

Reply:
Btw, I found https://github.com/apache/arrow-datafusion/blob/master/DEVELOPERS.md doesn't exist 😂, maybe it refers to CONTRIBUTING.md

Reply:
> https://github.com/apache/arrow-datafusion/blob/master/CONTRIBUTING.md#bootstrap-environment Can the relevant content in this file be deleted as well?

yeah